spec/lib/text_lines_filter_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
+
+describe SimpleCovLinterFormatter::TextLinesFilter do
+  let(:lines) do
+    [
+      "line1",
+      "line2",
+    ]
+  end
+
+  let(:filtered_lines) { "filtered_line1\nfiltered_line2" }
+
+  let(:expected_result) do
+    [
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Style/WordArray: Use %w or %W for an array of words.